alembic/versions/0d8aed449d2f_migration_message.py

```diff
@@ -0,0 +1,28 @@
+"""Migration message
+
+Revision ID: 0d8aed449d2f
+Revises: 2c4af13eef71
+Create Date: 2019-01-22 13:17:10.292648
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0d8aed449d2f'
+down_revision = '2c4af13eef71'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
+    # ### end Alembic commands ###
```

Comment:
You should have just one migration file, the one that defines the association table, kindly remove other unrelated alembic versions

Reply:
This has been worked on @cwizard2011. thanx for pointing it out.